llama-index-integrations/embeddings/llama-index-embeddings-bedrock/llama_index/embeddings/bedrock/base.py

```diff
@@ -389,7 +397,14 @@ def _get_request_body(
         if provider == PROVIDERS.AMAZON:
             if isinstance(payload, list):
                 raise ValueError("Amazon provider does not support list of texts")
-            request_body = json.dumps({"inputText": payload})
+            # Titan Embedding V2.0 has additional body parameters
+            if self.model_name == Models.TITAN_EMBEDDING_V2_0:
+                request_body = json.dumps(
+                    {"inputText": payload, **self.titan_body_kwargs}
+                )
```

Comment:
just ooc: would sending extra payload arguments lead to a Bad Request Error if passed with other models other than V2?

Reply:
@nerdai The only other Titan text embedding model is `Models.TITAN_EMBEDDING`, and if you pass those arguments to that it will throw a (somewhat cryptic) botocore error:
```
botocore.errorfactory.ValidationException: An error occurred (ValidationException) when calling the InvokeModel operation: Malformed input request: 2 schema violations found, please reformat your input and try again.
```

The implementation I've done will just ignore them if they aren't used for the Amazon Titan v2 model specifically.

Would the design prefer raising the exception?

Reply:
ah interesting. Can we just leave a brief comment then that including those arguments is problematic for the other text embedding model?

Reply:
Sure. Without raising the exception?